src/components/StatusDropdown/StatusDropdown.stories.tsx

```diff
@@ -0,0 +1,55 @@
+import { StatusDropdown } from '@components/StatusDropdown'
```

Comment:
tu też prettier nie zadziałał, puść autoformat

Reply:
ta, coś mi formatowanie na sejwie dla plików tsx w ui-kit nie działało to przeoczyłem parę rzeczy. Ogarnąłem już, musiałem w settings.json pogrzebać :P